src/vivarium_testing_utils/automated_validation/data_transformation/rate_aggregation.py

```diff
@@ -0,0 +1,20 @@
+from collections.abc import Callable
+from dataclasses import dataclass
+
+import pandas as pd
+
+from vivarium_testing_utils.automated_validation.data_transformation import utils
+from vivarium_testing_utils.automated_validation.data_transformation.data_schema import (
+    SingleNumericColumn,
+)
+
+
+@dataclass
+class RateAggregationWeights:
+    weight_keys: dict[str, str]  # Dataset keys needed
```

Comment:
document these properly. These things are attrs but kinda behave like properties from sphinx' perspective. [Here's how we decided to document dataclass attrs](https://github.com/ihmeuw/easylink/blob/437b83b0579af6a72370171f6034995165138190/src/easylink/graph_components.py#L29-L53)